### PR TITLE
remove reviewer ETA

### LINKED ISSUE #14917: Remove ETA from reviewer
Anki Desktop doesn't have an ETA in the reviewer and also have removed the ETA from the deck picker. So, following our ETA removal from the deck picker, it should be removed from the reviewer as well.

### PR DESCRIPTION
**Pull request by RobozinhoD**
## Purpose / Description
_Describe the problem or feature and motivation_

## Fixes
* Fixes #14917 

## Approach
I removed the eta code

## How Has This Been Tested?

I opened the reviewer

image

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/res/values/preferences.xml [hunk 1]
```diff
@@ -44,7 +44,6 @@
     <string name="pref_display_filenames_in_browser_key">card_browser_show_media_filenames</string>
     <string name="show_topbar_preference">showTopbar</string>
     <string name="show_progress_preference">showProgress</string>
-    <string name="show_eta_preference">showETA</string>
```
Comment (david-allison) [inline AnkiDroid/src/main/res/values/preferences.xml:47]: can `PreferenceUpgradeService` be updated to remove this from current collections

**Review by david-allison — CHANGES_REQUESTED** [review]
Minor request, rest LGTM

**Review by BrayanDSO — APPROVED** [review]

**Review by david-allison — APPROVED** [review]
Looks great, thanks. Will get in faster if you rebase onto main where CI is less flaky

**Comment by mikehardy** [comment]
I'm running strings through now

**Pull request merged**